pos_salesman_info/static/src/overrides/models/pos_store.js

```diff
@@ -0,0 +1,18 @@
+import { PosStore } from "@point_of_sale/app/store/pos_store";
+import { patch } from "@web/core/utils/patch";
+import { makeActionAwaitable } from "@point_of_sale/app/store/make_awaitable_dialog";
```

Comment:
```suggestion
import { PosStore } from "@point_of_sale/app/store/pos_store";
import { makeActionAwaitable } from "@point_of_sale/app/store/make_awaitable_dialog";
import { patch } from "@web/core/utils/patch";
```

Grouping imports from the same module together improves readability.

Reply:
done